scripts/config/pages.pl

```diff
@@ -42,6 +42,7 @@ sub all_pages {
         ['partnerships-contributor',   'static/partnerships/partnerships_contributor', 'full_width', 'Content Contributor programme'],
         ['partnerships-ib',            'static/partnerships/partnerships_ib',          'full_width', 'Introducing Broker'],
         ['liquidity-solutions',        'static/liquidity_solutions',                   'full_width', 'Liquidity Solutions'],
+        ['multi-account-manager',      'static/multi_account_manager',                 'full_width', 'Multiple Accounts Manager'],
```

Comment:
How about `multiple-accounts-manager`? (more similar to `h1` for SEO purpose as this page is not authenticated)